services/galley/src/Galley/API/Update.hs

```diff
@@ -1170,6 +1170,11 @@ removeMemberQualified lusr con qcnv victim =
       qcnv
       victim
 
+-- | if the public member leave api was called, we can assume that
+--   it was called by a user
+pattern EdMembersLeaveRemoved :: QualifiedUserIdList -> EventData
+pattern EdMembersLeaveRemoved l = EdMembersLeave EdReasonRemoved l
```

Comment:
Can't this just be a function?

Reply:
sure, it can be, but I thought this would be useful to mark it as a pattern so you can 
- use it both ways
- you show it's not doing anything extra (in contrast to e.g. a smart constructor) 

will change it if you wish.

Reply:
Nah, it's fine. Up to you.